### PR TITLE
feat: Add tracing concepts for failed requests

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
(still installing deps so will get linters/formatting to run local shortly)

File: src/sentry/static/sentry/app/utils/tracing.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+let spanId = null;
+let transactionId = null;
+
+import {uniqueId} from './guid';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/tracing.jsx]: lint will prob yell, imports need to be at top

File: src/sentry/tasks/post_process.py [hunk 2]
```diff
@@ -71,26 +70,6 @@ def check_event_already_post_processed(event):
     return not result
 
 
-def check_correct_event_sequencing(event):
```
Comment (mattrobenolt) [inline src/sentry/tasks/post_process.py]: Why is all of this being removed?

Comment (dcramer) [reply src/sentry/tasks/post_process.py]: looks like an issue w/ a rebase - fixing

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**